example/src/example_eco_handler.erl

```diff
@@ -0,0 +1,38 @@
+-module(example_eco_handler).
```

Comment:
That way the metadata can have info on the parameter this endpoint receives on PUT and the content-type it accepts and also on the data it returns on GET and the content-type it provides, therefore actually *using* trails for something.